cmd/manager/main.go

```diff
@@ -229,9 +232,11 @@ func addMetrics(ctx context.Context, cfg *rest.Config) error {
 		// ErrServiceMonitorNotPresent, which can be used to safely skip ServiceMonitor creation.
 		if err == metrics.ErrServiceMonitorNotPresent {
 			log.Info("Install prometheus-operator in your cluster to create ServiceMonitor objects", "error", err.Error())
+			return err
```

Comment:
Same here as in my earlier comment - we would only want to error if it's a case that the service monitor couldn't be created and it's for a reason other than "it already exists".